lib/Doctrine/ORM/Persisters/Entity/EntityPersister.php

```diff
@@ -190,6 +192,9 @@ public function getOwningTable($fieldName);
      *
      * @return object|null The loaded and managed entity instance or NULL if the entity can not be found.
      *
+     * @throws Exception
+     * @throws ORMException
```

Comment:
In which situations do we throw these exceptions and why should a caller handle them?